### PR TITLE
New helper function getIdentityPose()

### PR DESCRIPTION
**Pull request by davetcoleman**

### CODE REVIEW

**Comment by AndyZe** [comment]
Looks good to me. Would approve if it passes Travis. The error is:

`"'catkin_test_results' not found on the PATH. Please install catkin and source the environment to output the test result summary."`

File: src/rviz_visual_tools.cpp [hunk 1]
```diff
@@ -2754,6 +2754,22 @@ void RvizVisualTools::generateEmptyPose(geometry_msgs::Pose& pose)
...
 
+geometry_msgs::Pose RvizVisualTools::getIdentityPose()
+{
+  geometry_msgs::Pose pose;
+  // Position
+  pose.position.x = 0;
+  pose.position.y = 0;
+  pose.position.z = 0;
+
+  // Orientation on place
```
Comment (AndyZe) [inline src/rviz_visual_tools.cpp]: **on place** is a little confusing to me. Maybe just leave as `//orientation`

Comment (davetcoleman) [reply src/rviz_visual_tools.cpp]:
```suggestion
  // Orientation
```

**Comment by davetcoleman** [comment]
Fixed, lets see if Travis fixes itself

**Pull request merged**